stl/inc/iterator

```diff
@@ -814,6 +810,17 @@ template <input_iterator _Iter>
 struct iterator_traits<counted_iterator<_Iter>> : iterator_traits<_Iter> {
     using pointer = void;
 };
+
+template <contiguous_iterator _Iter>
+struct pointer_traits<counted_iterator<_Iter>> { // TODO: address LWG-3408 and include this

```

Comment:
We avoid `TODO` comments in headers because they're picked up by the IDE. This should be `TRANSITION`.